data/services/aws-rds.toml

```diff
@@ -1,3 +1,5 @@
+# Service metadata lives
+# https://github.com/18F/aws-broker/blob/59f949b/catalog-template.yml#L8
```

Comment:
I usually avoid branch URLs because then the line number can change or file can be moved and then the user is totally in the dark after clicking the link. Users can track a commit through it's history to find where the line moved to if they're looking for recent history.